R/generate_variability_terms.R

```diff
@@ -131,8 +133,15 @@ generate_ruv <- function(
   if (!is.null(seed)) {
     set.seed(seed)
   }
-  ruv$prop <- 1 + rnorm(nrow(ruv), 0, prop)
-  ruv$add <- rnorm(nrow(ruv), 0, add)
+  if (ltbs) {
```

Comment:
it's not complex code, but still would be better to rely on `PKPDsim::add_ruv` to avoid code duplication